srv/pg_monitor_demo_srv/dashboard/worker.py

```diff
@@ -0,0 +1,515 @@
+            {'date': '2021-02-20 13:00:00', 'value': 12732.0},
+            {'date': '2021-02-20 13:15:00', 'value': 12750.0},
+            {'date': '2021-02-20 13:30:00', 'value': 12689.0},
+            {'date': '2021-02-20 13:45:00', 'value': 12785.0},
+            {'date': '2021-02-20 14:00:00', 'value': 12730.0},
+            {'date': '2021-02-20 14:15:00', 'value': 12608.0},
+            {'date': '2021-02-20 14:30:00', 'value': 12102.0},
+            {'date': '2021-02-20 14:45:00', 'value': 11374.0},
+            {'date': '2021-02-20 15:00:00', 'value': 10955.0},
+            {'date': '2021-02-20 15:15:00', 'value': 10164.0},
+            {'date': '2021-02-20 15:30:00', 'value': 9299.0},
+            {'date': '2021-02-20 15:45:00', 'value': 8260.0},
+            {'date': '2021-02-20 16:00:00', 'value': 7178.0},
+            {'date': '2021-02-20 16:15:00', 'value': 6118.0},
+            {'date': '2021-02-20 16:30:00', 'value': 4966.0},
+            {'date': '2021-02-20 16:45:00', 'value': 3700.0},
+            {'date': '2021-02-20 17:00:00', 'value': 2049.0},
+            {'date': '2021-02-20 17:15:00', 'value': 751.0},
+            {'date': '2021-02-20 17:30:00', 'value': 484.0},
+            {'date': '2021-02-20 17:45:00', 'value': 174.0},
+            {'date': '2021-02-20 18:00:00', 'value': 17.0},
+            {'date': '2021-02-20 18:15:00', 'value': 0.0},
+            {'date': '2021-02-20 18:30:00', 'value': None},
+            {'date': '2021-02-20 18:45:00', 'value': None},
+            {'date': '2021-02-20 19:00:00', 'value': None},
+            {'date': '2021-02-20 19:15:00', 'value': None},
+            {'date': '2021-02-20 19:30:00', 'value': None},
+            {'date': '2021-02-20 19:45:00', 'value': None},
+            {'date': '2021-02-20 20:00:00', 'value': None},
+            {'date': '2021-02-20 20:15:00', 'value': 0.0},
+            {'date': '2021-02-20 20:30:00', 'value': None},
+            {'date': '2021-02-20 20:45:00', 'value': None},
+            {'date': '2021-02-20 21:00:00', 'value': None},
+            {'date': '2021-02-20 21:15:00', 'value': None},
+            {'date': '2021-02-20 21:30:00', 'value': None},
+            {'date': '2021-02-20 21:45:00', 'value': None},
+            {'date': '2021-02-20 22:00:00', 'value': None},
+            {'date': '2021-02-20 22:15:00', 'value': None},
+            {'date': '2021-02-20 22:30:00', 'value': None},
+            {'date': '2021-02-20 22:45:00', 'value': None},
+            {'date': '2021-02-20 23:00:00', 'value': None},
+            {'date': '2021-02-20 23:15:00', 'value': None},
+            {'date': '2021-02-20 23:30:00', 'value': None},
+            {'date': '2021-02-20 23:45:00', 'value': None}
+        ]
+    }
+}
+
+
+solar_edge_site2 = {
+    'energy': {
+        'timeUnit': 'QUARTER_OF_AN_HOUR',
+        'unit': 'Wh',
+        'measuredBy': 'INVERTER',
+        'values': [
+            {'date': '2021-02-20 00:00:00', 'value': None},
```

Comment:
[Question] Would it be correct that I display `none` values (received as `null` values in the API response) as `0.0` values? I haven't tried passing this data to the charts yet. Might be the case the charts parse these values correctly without me having to parse the data first. 

Should the front-end display some kind of error in this case, or receiving `null` values is expected behaviour?

Reply:
Receiving `null` is expected behavior, as it represents that there is no data available for that time. Typically, this represents the solar power generation equipment being turned off.
For the purposes of the display, they can be safely shown as `0.0`.